_posts/2019-06-18-snowplow-snowflake-loader-0.5.0-released/index.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: Snowplow Snowflake Loader 0.5.0 released
+title-short: Snowflake Loader 0.5.0
+tags: [snowflake, storage, relational databases]
+author: Enes
+category: Releases
+permalink: /blog/2019/06/18/snowplow-snowflake-loader-0.5.0-released/
+---
+
+We are pleased to announce version 0.5.0 of the [Snowplow Snowflake Loader][snowflake-loader-repo]! This release introduces bad row support with its [new format][new-bad-row-format-rfc].
+
+Read on below the fold for:
+
+1. [Bad row support](#bad-row)
+2. [Upgrading](#upgrading)
+3. [Getting help](#help)
+
+<!--more-->
+
+<h2 id="bad-row">1. Bad Row Support</h2>
+
+Data quality has always been a huge focus for Snowplow. Being a non-lossy data pipeline is one of the central pieces of this data quality puzzle. Indeed, other pieces such as data validity (through our schema validation technology) or data richness (through our enrichments) connect directly with the non-lossiness piece.
+
+In practice, for a Snowplow pipeline, non-lossiness meant that when something went wrong anywhere in the pipeline, instead of discarding it, the data impacted was parked as “bad” for later inspection, fixing and reprocessing. In Snowplow jargon, this bad data is called “bad rows”.
+
+With this release, we are adding bad row support to Snowflake Transformer with [its new format][new-bad-row-format-rfc]. From now on, Snowflake Transformer will continue to run when it encounters unexpected data and it will write the bad rows to a separate place for further inspection instead of halting with exception. Snowflake Loader is the first component of the pipeline introducing bad rows and enrichment is about to be next one very soon.
```

Comment:
The last sentence in this section is a bit unclear, does it mean bad rows will soon be introduced to enrichments as well? In this case, it would be better to say "Snowflake Loader is the first component of the pipeline introducing bad rows, and soon enrichment will also include this feature." or something along these lines.